dipy/denoise/tests/test_gibbs.py

```diff
@@ -0,0 +1,151 @@
+import numpy as np
+from dipy.denoise.gibbs import (_gibbs_removal_1d, _gibbs_removal_2d,
+                                gibbs_removal, _image_tv)
+from numpy.testing import (assert_, assert_array_almost_equal, assert_raises)
+
+# Produce a 2D image
```

Comment:
Can you move this part to a `setup_module()` function like [here](https://github.com/nipy/dipy/blob/master/dipy/reconst/tests/test_fwdti.py#L23). Do not forget to explicit what is global or not

Reply:
Very nice suggestion!